src/main/java/net/authorize/sample/SampleCode.java

```diff
@@ -134,19 +135,20 @@ private static void ShowMethods()
         System.out.println("    GetAnAcceptPaymentPage");
     }
 
-    private static void RunMethod(String methodName)
+    private static void RunMethod(String methodName) throws IllegalArgumentException, IllegalAccessException, NoSuchFieldException, SecurityException
```

Comment:
This change is unacceptable. It will break existing Travis CI flows

Reply:
Exceptions and throws removed form method definition.